### ISSUE TITLE
Confusing & broken Inheritance - Nightly 1.77.2

### ISSUE THREAD

**Issue by pauldurham** [mRemoteNG/mRemoteNG#2152]
The items below are incredibly confusing without proper help text, and don't seem to work as expected:
- Inheritance
- Default Properties
- Default inheritance

It is not obvious how to use the above functions.  Also, the default inheritance doesn't appear to work.

## Expected Behavior
The above items should allow the user to set default properties & inheritance and then turn on inheritance, but they don't seems to work properly.  This is assuming I am even configuring the settings correctly, which I may not be doing because it is confusing.

## Current Behavior
Even if I set default inheritance, it seems the settings are not being pulled through.

## Context
With servers in folders, it want to be able to use the same login credentials, for example.

## Your Environment
Windows 10 Enterprise 64-bit 20H2.

**Comment by simonai1254** [comment]
I completely agree with this point, we already had a discussion about the inheritance in regards with the Clipboard Settings in Issue #1968.

**Comment by LIyoai** [comment]
#975

**Comment by robertpopa22** [comment]
**Fix available for testing**

A fix for this issue has been implemented in our community fork (`robertpopa22/mRemoteNG`, branch `main`). A beta build including this fix is available from the fork's Releases page.

Please test and report if this resolves your issue.

---

> *We sincerely appreciate the mRemoteNG maintainers and the wonderful community that has kept this project alive and thriving for over a decade. Your work powers thousands of IT professionals worldwide, including us. Our fork builds entirely on your foundation — we're preparing a consolidated Pull Request to contribute these improvements back to the official project. Thank you for everything you've built.*
>
> *— robertpopa22, community contributor*
